src/components/Dashboard.js

```diff
@@ -333,13 +340,13 @@ export class DashboardPage extends React.Component {
         return {
           ...student,
           qualification: qualificationKeys[student.qualification],
-          altNumber:contacts ? contacts.alt_mobile : contacts,
+          altNumber: contacts ? contacts.alt_mobile : contacts,
         };
       });
-      if(studentData.length > 0){
-        studentData[0] =  {... studentData[0], ...obj}
+      if (studentData.length > 0) {
```

Comment:
Probably has worse readability, but you can use `studentData.length` here instead of `studentData.length > 0` because `0` is a falsey value.